src/server/_security.py

```diff
@@ -22,28 +22,21 @@
 API_KEY_SOFT_WARNING = API_KEY_HARD_WARNING - timedelta(days=14)
 
 # rollout warning messages
-# intended usage: in place of API_KEY_WARNING_TEXT
-# phase 1 / soft warning: ROLLOUT_WARNING_RATE_LIMIT or ROLLOUT_WARNING_MULTIPLES
-# phase 2 / hard warning: (ROLLOUT_WARNING_RATE_LIMIT + PHASE_2_STOPGAP) or (ROLLOUT_WARNING_MULTIPLES + PHASE_2_STOPGAP)
-
-ROLLOUT_WARNING_RATE_LIMIT = "This request exceeded the anonymous limit on requests per minute."
-ROLLOUT_WARNING_MULTIPLES = "This request exceeded the anonymous limit on selected multiples."
-_ROLLOUT_WARNING_AD_FRAGMENT = "To be exempt from this limit, authenticate your requests with an API key, which will be enforced starting {}. Registration now available at {}.".format(
-    API_KEY_REQUIRED_STARTING_AT, API_KEY_REGISTRATION_FORM_LINK_LOCAL
-)
+ROLLOUT_WARNING_RATE_LIMIT = "This request exceeded the anonymous limit on requests per hour, which will be enforced starting {}.".format(API_KEY_REQUIRED_STARTING_AT)
```

Comment:
maybe remove the verbiage that refers to the time bucket for the rate?  its not likely to change within the 4-week period where this message will be shown, so it probably doesnt make too much of a difference.
```suggestion
ROLLOUT_WARNING_RATE_LIMIT = "This request exceeded the rate limit on anonymous requests, which will be enforced starting {}.".format(API_KEY_REQUIRED_STARTING_AT)
```